mobile/lib/src/presentation/navigation/routes/router.gr.dart

```diff
@@ -0,0 +1,113 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
```

Comment:
Don't track generated files

Add `*.gr.dart` rule to gitignore

Reply:
Add `flutter packages pub run build_runner build --delete-conflicting-outputs` to  `.github/workflows/mobile.yml`

Add `flutter packages pub run build_runner watch`  to readme